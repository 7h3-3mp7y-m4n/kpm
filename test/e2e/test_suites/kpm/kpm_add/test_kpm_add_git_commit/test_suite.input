kpm add -git https://github.com/KusionStack/catalog.git -commit a29e3db