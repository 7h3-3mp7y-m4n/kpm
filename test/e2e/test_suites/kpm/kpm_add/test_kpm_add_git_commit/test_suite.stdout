adding dependency 'catalog'
cloning 'https://github.com/KusionStack/catalog.git' with commit 'a29e3db'